src/Console/Command/Migrate.php

```diff
@@ -27,16 +27,23 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'Environment name'
             )
+            ->addOption(
+                '--configuration',
```

Comment:
move this to `configure ` bethod in abstract command because this option must be configuted in all commands

Reply:
okay